addons/skin.estuary/language/resource.language.en_gb/strings.po

```diff
@@ -911,3 +911,9 @@ msgstr ""
 msgctxt "#31611"
 msgid "System"
 msgstr ""
+#: /xml/SkinSettings.xml
+#. Label to show TV?Radio menu action
```

Comment:
radio? this currently only applies to live tv, or am I wrong? So how about `Label for the "Home screen menu action for live TV" setting`?

Reply:
The behavior should customizable for both radio and Live TV.

Reply:
@ksooo I can do both but I would like radio to default to Channels if that is ok, since people tend to listen to radio by station and not by show.  Plus it is pretty much only dvb that has EPG for radio.

Reply:
not even DVB has proper radio EPG, at least not on my end

Reply:
> not even DVB has proper radio EPG

For me, most of the German radio stations have. DVB-S, Astra.

Reply:
@emveepee one of the guys in the forum mentioned, that he want Channels, because radio stations have no EPG for him. I assumed, he would like Guide as default for TV, but Channels for Radio. Might be similar for other people, so I think having two settings, one for TV one for Radio makes sense.

Reply:
> EDIT: Okay, you already confirmed two settings would be fine for you. :-) No hard feelings about the radio default as long as default for TV stays Guide.

I don't mind Guide as default, I know how to change it, it is more a support issue I think.   I was actually using it inefficiently before, hitting right then click so I save a step.